rotkehlchen/constants/assets.py

```diff
@@ -2209,3 +2209,15 @@
     protocol=None,
 )
 CONSTANT_ASSETS.append(A_ENS)
+A_PSP = EthereumToken.initialize(
+    address=string_to_ethereum_address('0xcAfE001067cDEF266AfB7Eb5A286dCFD277f3dE5'),
+    decimals=18,
+    name="ParaSwap",
```

Comment:
even though this will be re-generated later by the automatic script so this does not matter much, but in general we use single quoted.

```suggestion
    name='ParaSwap',
```